common/protocols/src/main/proto/app/coronawarn/server/common/protocols/external/exposurenotification/diagnosis_key_batch.proto

```diff
@@ -21,6 +21,7 @@ message DiagnosisKey {
   // E.g. 2 means TEK is 2 days after onset of symptoms.
   uint32 days_since_onset_of_symptoms = 8;
 }
+
```

Comment:
This is harmless to the protobuf spec, but useless in this PR. So could we remove to avoid future conflicts?